alphabet/alphabet_contract.go

```diff
@@ -37,10 +36,12 @@ func _deploy(data interface{}, isUpdate bool) {
 	if isUpdate {
 		args := data.([]interface{})
 		common.CheckVersion(args[len(args)-1].(int))
+		storage.Delete(ctx, notaryDisabledKey)
```

Comment:
In theory, everything done by `InitVote` as well as any "in-progress" votes should also be removed.
For us it makes no sense because all the networks we use are already notary-enabled. But I would at least mention this in the commit message.